e2e/integration/searchtests.js

```diff
@@ -11,12 +11,12 @@ context( 'Test search', () => {
     } );
 
     it( 'Test for valid search count', () => {
-        cy.get( '.opc-nav-search__btn' ).click( { force: true } )
-        cy.wait(2000)
-        cy.get( 'input[name="query"]' ).type( 'Feedback', { force: true } );
-        cy.wait(500)
-        cy.get( '.opc-nav-search__btn' ).click();
-        cy.get( '.pf-u-mt-xl' ).should( 'include.text','results found')
+        cy.wait(150)
+        cy.get( 'input[name="query"]', { includeShadowDom: true } ).click( { force: true } ).clear( { force: true } ).type( 'Feedback', { force: true } );
+        cy.get( '.op-search__btn' ).click({force:true});
+        //cy.wait(500)
+        //cy.get( '.opc-nav-search__btn' ).click({force:true});
```

Comment:
```suggestion
```

Reply:
HI @riginoommen 
issue resolved in https://github.com/1-Platform/one-platform/pull/1331/commits/17a99be5934e00bcc6baa13bf303989fb9e18afd